onnxruntime/core/providers/webnn/builders/helper.cc

```diff
@@ -53,9 +53,12 @@ bool IsInputSupported(const NodeArg& input, const std::string& parent_name, cons
   }
 
   for (const auto& dim : shape_proto->dim()) {
-    // For now we workaround dynamic shape support by assuming 1.
+    // WebNN doesn't support dynamic shape, use sessionOptions.freeDimensionOverrides to fix the shape.
```

Comment:
```suggestion
    // WebNN doesn't support dynamic shape - use sessionOptions.freeDimensionOverrides to fix the shape.
```

(grammar nit about run-on sentences, to use hyphen dash, semicolon, or period, since there are both complete clauses rather than phrases)